docs/source/guides/getting_started.rst

```diff
@@ -380,7 +401,50 @@ the implementation of your service. What we are able to do is build a client who
 on the same JAX-RS annotations you use for your server. That makes it really easy to make clients
 for your Java consumers; however, it won't help you much with consumers of other languages. Instead
 of trying to explain all the concepts that are going on within the client we recommend you take a
-look at our `client documentation <https://github.com/cerner/beadledom/tree/master/client>`_.
+look at our `client documentation <https://github.com/cerner/beadledom/tree/master/client>`_..
+
+HelloWorldResource
+++++++++++++++++++
+
+.. code-block:: java
+
+  @Path("/hello")
+  public interface HelloWorldResource {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    GenericResponse<HelloWorldDto> getHelloWorld();
+  }
+
+This is the definition for the ``/hello`` resource from a client-side perspective. This interface
+allows automatically generating a client implementation via a Java proxy object.
+
+.. code-block:: java
+
+  /**
+   * JAX-RS annotation defining the base URI path for the resource.
+   */
+  @Path("/hello")
+  public interface HelloWorldResource {
+
+    /**
+     * JAX-RS annotation for the HTTP verb this method is providing functionality for.
+     */
+    @GET
+
+    /**
+     * JAX-RS annotation for the media type our service returns.
+     */
+    @Produces(MediaType.APPLICATION_JSON)
+
+    /**
+     * Method definition for the resource. The GenericResponse is an implementation provided
```

Comment:
Maybe note for responses that will not include a body in the response (DELETE for instance) it's best to continue using the regular JAX-RS `Response` class.

Reply:
https://github.com/cerner/beadledom/pull/133/commits/90d88965913cd8d3fcb84c756c7d2516e8aa2506